execinquery.go

```diff
@@ -49,53 +44,80 @@ func run(pass *analysis.Pass) (interface{}, error) {
 				return
 			}
 
-			var i int
+			replacement := "Exec"
+			var i int // the index of the query argument
 			if strings.Contains(selector.Sel.Name, "Context") {
+				replacement = "ExecContext"
 				i = 1
 			}
 
-			var s string
-			switch arg := n.Args[i].(type) {
-			case *ast.BasicLit:
-				s = strings.Replace(arg.Value, "\"", "", -1)
-
-			case *ast.Ident:
-
-				switch arg2 := arg.Obj.Decl.(type) {
-				case *ast.AssignStmt:
-					for _, stmt := range arg2.Rhs {
-						basicLit, ok := stmt.(*ast.BasicLit)
-						if !ok {
-							continue
-						}
-
-						s = strings.Replace(basicLit.Value, "\"", "", -1)
-					}
-				case *ast.ValueSpec:
-					basicLit, ok := arg2.Values[0].(*ast.BasicLit)
-					if !ok {
-						return
-					}
-
-					s = strings.TrimLeftFunc(basicLit.Value, func(r rune) bool {
-						return !unicode.IsLetter(r) && !unicode.IsNumber(r)
-					})
-					s = strings.Replace(s, "\"", "", -1)
-				}
-
-			default:
+			if len(n.Args) <= i {
 				return
 			}
 
-			if strings.HasPrefix(strings.ToLower(s), "select") {
+			query := getQueryString(n.Args[i])
+			if query == "" {
 				return
 			}
 
-			s = strings.ToTitle(strings.SplitN(s, " ", 2)[0])
+			query = strings.TrimSpace(cleanValue(query))
+			parts := strings.SplitN(query, " ", 2)
+			cmd := strings.ToTitle(parts[0])
```

Comment:
```suggestion
			cmd := strings.ToUpper(parts[0])
```
Final nitpicking---let's just use the obvious function to kill any possible misunderstanding. :stuck_out_tongue: